device/juniper/x86_64-juniper_qfx5210-r0/plugins/sfputil.py

```diff
@@ -5,13 +5,15 @@
     from sonic_sfp.sfputilbase import *
     import sys
     import os
+    import io
     import string
     from ctypes import create_string_buffer
-    # sys.path.append('/usr/local/bin')
-    # import sfp_detect
+    from sonic_daemon_base.daemon_base import Logger
 except ImportError, e:
     raise ImportError (str(e) + "- required module not found")
 
+SYSLOG_IDENTIFIER = "xcvrd"
```

Comment:
Why syslog identifier of "xcvrd"? Identifier should be specific to your plugin.

Reply:
Updated.